src/target/riscv/riscv.c

```diff
@@ -188,6 +188,8 @@ int riscv_reset_timeout_sec = DEFAULT_RESET_TIMEOUT_SEC;
 bool riscv_use_scratch_ram;
 uint64_t riscv_scratch_ram_address;
 
+bool riscv_havereset_not_supported;
```

Comment:
what is the default of this?

Reply:
The default is false, which is why I put a negation in the variable name.

So the default behavior is for OpenOCD to assume havereset is implement.